core/templates/dev/head/editor/ExplorationHistory.js

```diff
@@ -53,6 +54,32 @@ oppia.controller('ExplorationHistory', [
     }
   });
 
+  $scope.submitVersion = function($event, versionNumber) {
```

Comment:
Suggest renaming this function something like "changeSelectedVersions". This is called when checked checkboxes are changed, which seems to be a different action from submitting versions to compare (when the "Compare Selected Versions" button is clicked).

I'm not entirely sure about this though, since the versions are actually sent to the comparison function in here. Maybe it will make more sense to submit both versions when the button is clicked? (Originally, versions were submitted as the radio buttons were changed so that users could change the compared versions without clicking the button again, but now this is no longer possible.)